ioctl/cmd/account/account.go

```diff
@@ -178,6 +178,25 @@ func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
 	return response.AccountMeta, nil
 }
 
+// IsSignerExist checks whether signer account is existed
+func IsSignerExist(signer string) bool {
+	addr, err := address.FromString(signer)
+	if err != nil {
+		return false
+	}
+	// find the account in keystore
+	ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+		keystore.StandardScryptN, keystore.StandardScryptP)
+	for _, ksAccount := range ks.Accounts() {
+		if address.Equal(addr, ksAccount.Address) {
+			return true
+		}
+	}
+	// find the account in pem files
+	_, err = findSm2PemFile(addr)
```

Comment:
I still don't see why we need a mixture of sm2 and current pems. Sm2 account format is not determined by user input, https://github.com/iotexproject/iotex-core/blob/1e3895b75321f0ac8b946621958923c08d518fac/ioctl/cmd/account/accountcreateadd.go#L47. "xctl" always and only creates or uses sm2 accounts.

Overall, I think you implementation is misleading to users of "xctl", who shouldn't be aware of the exists of two different account systems or algorithms.